src/Features/Core/Diagnostics/EngineV1/DiagnosticIncrementalAnalyzer.cs

```diff
@@ -86,7 +87,8 @@ public override Task DocumentResetAsync(Document document, CancellationToken can
                 // we remove whatever information we used to have on document open/close and re-calcuate diagnostics
                 // we had to do this since some diagnostic analyzer change its behavior based on whether the document is opend or not.
```

Comment:
Is this comment duplicated on lines 72 and 87? It seems that only 1 occurrence is fixed.